solr/core/src/test/org/apache/solr/cli/SolrProcessManagerTest.java

```diff
@@ -84,9 +83,11 @@ private static int findAvailablePort() throws IOException {
   private static Pair<Integer, Process> createProcess(int port, boolean https) throws IOException {
     // Get the path to the java executable from the current JVM
     String classPath =
-        Arrays.stream(System.getProperty("java.class.path").split(File.pathSeparator))
+        Arrays.stream(
+                System.getProperty("java.class.path").split(System.getProperty("path.separator")))
```

Comment:
I'm suprised we don't have sort of nicer way of doing this than `System.getProperty`..  Is this common enough that we should have a helper?

Reply:
Is "FileSystems.getDefault().getSeparator();" a possiblity?

Reply:
I'm good with your replacement, albeit it'd be slightly nicer to declare a variable to hold the value as you're going to refer to it multiple times.  NIO has no alternative.  It's easier to just ban File and use this more awkward lookup because it's so rare to actually use this.